docs/sinco/v3.x/2_tutorials/2_page/5_waiting.md

```diff
@@ -0,0 +1,75 @@
+# Waiting
+
+## Table of Contents
+
+- [Before You Get Started](#before-you-get-started)
+- [Folder Structure End State](#folder-structure-end-state)
+- [Steps](#steps)
+- [Verification](#verification)
+
+## Before You Get Started
+
+`Page` provides methods to wait in specific scenarios, such as waiting for the
+page to change after clicking a button, or waiting for an anchor change on the
+uri. These help in ensuring your following code matches what the page should be,
+so say for example, you click a button that changes the page after 5 seconds.
+The `waitForPageChange()` method will wait for this, so your other actions and
+assertions aren't trying to run whilst the new page hasn't loaded yet.
+
+The following methods Sinco provides are:
+
+- `waitForPageChange()`
+- The `waitForPageChange()` method will wait for the page to change, eg clicking
+  an anchor tag that directs the user to a new uri.
```

Comment:
```suggestion
Sinco provides the following metod:

- `waitForPageChange()`: This will wait for the page to change (e.g., clicking
  an anchor tag that directs the user to a new URI).
```